presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/serializers/StringRowSerializer.java

```diff
@@ -410,7 +410,7 @@ else if (type.equals(VARCHAR)) {
             return (T) new String(value);
         }
         else {
-            throw new PrestoException(NOT_SUPPORTED, "StringLexicoder does not support decoding type " + type);
+            throw new PrestoException(NOT_SUPPORTED, "StringRowSerializer does not support decoding type " + type);
```

Comment:
It is not a typo. Maybe  just `"Not supported decoding type: " + type`?

Reply:
Done